src/NeoService/Settings.cs

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2015-2023 The Neo Project.
+//
+// The Neo.Network.RPC is free software distributed under the MIT software license,
+// see the accompanying file LICENSE in the main directory of the
+// project or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Microsoft.Extensions.Configuration;
+using Neo.SmartContract.Native;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+
+namespace Neo.Plugins
+{
+    class Settings
+    {
+        public IReadOnlyList<NeoServiceSettings> Servers { get; init; }
+
+        public Settings(IConfigurationSection section)
+        {
+            Servers = section.GetSection(nameof(Servers)).GetChildren().Select(p => NeoServiceSettings.Load(p)).ToArray();
+        }
+    }
+
+    public class NeoServiceSettings
+    {
+        public uint Network { get; init; }
+        public IPAddress BindAddress { get; init; }
+        public ushort Port { get; init; }
+        public string SslCert { get; init; }
+        public string SslCertPassword { get; init; }
+        public string[] TrustedAuthorities { get; init; }
+        public int MaxConcurrentConnections { get; init; }
```

Comment:
I think that NeoService should not have http/socket related methods

Reply:
it can be disabled by default, wont hurt existing service, but will be useful for those who need it.

Reply:
If we want to separate it to reuse it between wss and rpc we have to do it in a way that an http protocol is not necessary, perhaps in the future we will make another type of module that reuses NeoService without being related to http. Otherwise it is better to extend RpcServer and use wss.

Reply:
i am confused, this one is only about APIs, rpc is in rpcserver. neoservice itself  does not contain http.

Reply:
ok i see, if those fields is your concern, we can remove them back to the rpcserver